bot.py

```diff
@@ -389,6 +391,32 @@ def __init__(self, *args, choices=None, opts_to_inv=None, **kwargs):
         )
         self.add_item(select_menu)
 
+class EmojiSyncView(discord.ui.View):
+    # Might be able to delete guild reference
+    def __init__(self, emoji, mod_type: str, old_name=None, *args, **kwargs):
+        super().__init__(timeout=None, *args, **kwargs)
+        self.emoji = emoji
+        self.old_name = old_name
+        
+        # Mod type will be a String of 'Add', 'Del', or 'Name' 
+        self.mod_type = mod_type
+
+    @discord.ui.button(label='Accept', style=discord.ButtonStyle.green)
+    async def accept_callback(self, button, interaction: discord.Interaction):
+        if self.mod_type == 'Add':
+            await sync_add(bot=bot, emoji=self.emoji)
+        elif self.mod_type == 'Del':
+            await sync_delete(bot=bot, emoji=self.emoji)
+        else:
+            await sync_name(bot=bot, old_name=self.old_name, new_emoji=self.emoji)
+
+        await interaction.response.edit_message('Okay! I will sync this now.', view=None, ephemeral=True)
```

Comment:
Does this work? Has the interaction passed to this button press been responded to already? Or does this just change the view message contents?

I haven't tested editing response like this before myself, but if it works that's cool.

Reply:
Based on what I have read it should change the appearance of the current message. I have not been able to test it myself though as @tjhubz has been testing it